kubetest/main.go

```diff
@@ -120,6 +120,7 @@ type options struct {
 	up                      bool
 	upgradeArgs             string
 	boskosWaitDuration      time.Duration
+	useKrel                 bool
```

Comment:
Why we need this `useKrel` in the `options` struct, if we use one from `stageStrategy`?

Reply:
I'm not sure if I get that correctly. This option is directly set from the CLI. Do you see room for improvements here?

Reply:
We use `&o.stage.useKrel` as a variable for storing the flag's value, so I was thinking do we need `o.useKrel` or just `o.stage.useKrel` is enough.

Although, I don't any experience with kubetest, so your way might be the correct one actually. :)

Reply:
Yes, you're right, I'll remove that flag. Thanks for the explanation! :pray: